Formula/aescrypt.rb

```diff
@@ -3,6 +3,12 @@ class Aescrypt < Formula
   homepage "https://aescrypt.sourceforge.io/"
   url "https://aescrypt.sourceforge.io/aescrypt-0.7.tar.gz"
   sha256 "7b17656cbbd76700d313a1c36824a197dfb776cadcbf3a748da5ee3d0791b92d"
+  license "BSD-4-Clause"
+
+  livecheck do
+    url :homepage
+    regex(/aescrypt[._-]v?(\d+(?:\.\d+)+)\.t/i)
```

Comment:
```suggestion
    regex(/href=.*?aescrypt[._-]v?(\d+(?:\.\d+)+)\.t/i)
```

When identifying versions from files on an HTML page, we generally restrict matching to `href` attributes.